packages/api/src/external/commonwell/patient-shared.ts

```diff
@@ -33,7 +33,7 @@ export class PatientDataCommonwell extends PatientExternalDataEntry {
   constructor(
     public patientId: string,
     public personId?: string | undefined,
-    public status?: LinkStatus | undefined,
+    public discoverStatus?: LinkStatus | undefined,
```

Comment:
CW doesn't have the concept of Patient Discovery, we've been calling it LinkStatus (thus the type and other functions like `getLinkStatusCW`, even some internal endpoints).

I suggest we leave as is and revisit this when we're revamping the patient state/status. Unless you want to make the case for this update and we can discuss (happy to take this to Slack to speed it up).